config/locales/en.yml

```diff
@@ -59,6 +61,8 @@ en:
         change: Change your password
         forgot: Forgot password?
         update: Update your password
+      registrations:
+        new: Email address
```

Comment:
Seems like a strange heading?    I'd prefer it if the name of the translation string was a little more meaningful, although I guess you matching the name of the action here.